First draft of layout of inventory menu

sweeneyde opened the pull request.

sweeneyde committed:
First draft of layout of inventory menu

sweeneyde committed:
Merge branch 'master' into inventory

bhasseman reviewed (approved):
Compiles and runs well. Enter key opens the inventory, correctly displays where Link is on the map. I really like the code/logic for the InventoryState class, well done with that.
LGTM

Commit:
HUD and Inventory Setup

Added graphics for all hud features and implemented osme inventory management to allow for it

jfc978 commented:
Added HUD and Inventory improvements, built on initial setup from this branch

sweeneyde commented:
The HUD looks good so far. I'm good to merge this.
